packages/lime-system/files/usr/lib/lua/lime/network.lua

```diff
@@ -111,8 +111,10 @@ function network.setup_rp_filter()
 	
 	sysctl_options = sysctl_options .. "net.ipv4.conf.default.rp_filter=2\nnet.ipv4.conf.all.rp_filter=2\n";
 	sysctl_file = io.open(sysctl_file_path, "w");
-	sysctl_file:write(sysctl_options);
-	sysctl_file:close();
+	if sysctl_file ~= nil then
```

Comment:
Shouldn't it do something if the file doesn't exist? like notify or something?

Reply:
I wrote it this way because I think that this should only happen inside the unit testing environment (not having permissions to write to /etc) and we don't want to get a notification in this case. I can add a comment to clarify the intention as I think like you that it is unexpected to read that without an error report.

Reply:
The if statement looks a little strange, that's all.